### DISCUSSION TITLE
VLAN group scope not doing anything?

### DISCUSSION THREAD

**Discussion by spamik** [netbox-community/netbox#8523] [Q&A]
Hi,

I'm playing with netbox 3.1.6 and I found out something which didn't make sense for me. Situation:
- I create two sites: "Site A" and "Site B"
- I create two VLAN groups: "Site A VG" with site scope to "Site A" and "Site B VG" with site scope to "Site B"
- create some vlans under each VLAN group

Now, first, If I open any site, in statistics I have 0 VLANs. I believe that I should see number of VLANs assigned through VLAN group but it's only statistic, doesn't matter...

But now when I create device in "Site A", I create interface and select 802.1Q Mode Access. Now if I look under to VLAN groups I can see both VLAN groups even the one with scope for "Site B".

Is this expected behaviour or it's some bug? Because if it's expected I don't understand what is purpose of scope in VLAN group as when I'm editing interface I can see all VLAN grouped and not only that in scope where my device reside in.

**Comment by foxx** [comment]
I'm having the same issue. If I create a VLAN Group and set the scope, it does not show up in Related Objects.
